prow/plugins/plugins.go

```diff
@@ -369,6 +369,14 @@ type Lgtm struct {
 	// StoreTreeHash indicates if tree_hash should be stored inside a comment to detect
 	// squashed commits before removing lgtm labels
 	StoreTreeHash bool `json:"store_tree_hash,omitempty"`
+	// WARNING: Read below for potential risk before you enable this option.
```

Comment:
nit: This warning should probably be moved down 1 line so that the GoDocs still work properly, otherwise LGTM.

Reply:
Does this flag really require a miniature novel worth of documentation?

Reply:
I personally don't think so, but I respect the desire to call out the risk of stickly LGTM.